pkg/fanal/analyzer/analyzer.go

```diff
@@ -286,6 +287,11 @@ func NewAnalyzerGroup(groupName Group, disabledAnalyzers []Type) AnalyzerGroup {
 		group.configAnalyzers = append(group.configAnalyzers, a)
 	}
 
+	// load classifier when scanning licenses
+	if !slices.Contains(disabledAnalyzers, TypeLicenseFile) {
+		licensing.NewClassifier()
+	}
```

Comment:
`NewClassifier` should return a new struct `Classifier` and it should be called in license analyzer.